src/gc/sample/gcenv.ee.cpp

```diff
@@ -175,18 +175,21 @@ bool GCToEEInterface::RefCountedHandleCallbacks(Object * pObject)
     return false;
 }
 
-bool GCToEEInterface::IsPreemptiveGCDisabled(Thread * pThread)
+bool GCToEEInterface::IsPreemptiveGCDisabled()
 {
+    Thread* pThread = ::GetThread();
```

Comment:
It seems like ::GetThread() may return nullptr so it would be good to add assert(pThread != nullptr);

Reply:
This is in the GC sample and the sample [never returns null](https://github.com/dotnet/coreclr/blob/master/src/gc/sample/gcenv.ee.cpp#L113). In general though, you're right - the VM version of `gcenv.ee.cpp` has to be robust in the face of a null `GetThread`.

Reply:
I see. I missed it.